modules/ldap-auto-syncing.adoc

```diff
@@ -0,0 +1,246 @@
+// Module included in the following assemblies:
+//
+// * authentication/ldap-syncing.adoc
+
+[id="ldap-auto-syncing_{context}"]
+= Automatically syncing LDAP groups
+
+You can automatically sync LDAP groups on a periodic basis by configuring a cron job.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have configured an LDAP identity provider (IDP), which involved creating an LDAP secret and config map.
+
+// TODO Any other prereqs? what about anything else w/ the sync stuff, like a config file, etc? Looks like they embed it in the config map
+
+.Procedure
+
+. Create a namespace where the cron job will run:
++
+[source,terminal]
+----
+$ oc create namespace ldap-sync <1>
+----
+<1> This procedure uses a namespace called `ldap-sync`.
+
+. Copy the secret and config map that you created when configuring the LDAP identity provider to this new namespace.
++
+The secret and config map exist in the `openshift-config` namespace and must be copied to the new `ldap-sync` namespace.
++
+// TODO: Provide actual instructions on how to copy, or just leave it like this?
+
+. Define a service account:
++
+.Example `ldap-sync-service-account.yaml`
+[source,yaml]
+----
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: ldap-group-syncer
+  namespace: ldap-sync
+  labels:
+    app: cronjob-ldap-group-sync
+----
++
+// TODO: Mention the significance of this label that is used across all objects. Is it required? Is it also needed on the config map and secret from when setting up the LDAP IDP?
+
+. Create the service account:
++
+[source,terminal]
+----
+$ oc create -f ldap-sync-service-account.yaml
+----
+
+. Define a cluster role:
++
+.Example `ldap-sync-cluster-role.yaml`
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: ldap-group-syncer
+  labels:
+    app: cronjob-ldap-group-sync
+rules:
```

Comment:
@stlaz Are these rules exactly what they would need to set? Or do we expect any to be adjusted / are any worth calling out?

Reply:
I am not sure about the `''` apiGroup but otherwise this seems ok

Reply:
@stlaz Do you think we should remove that `''` group?

Reply:
That should be there to denote `core` or `v1` group, which isn't named. Although most frequently I've seen `""` being used, although from checking it shouldn't matter since both will work just fine.

Reply:
Thanks, I'll leave it in